src/effector/__tests__/domain.test.ts

```diff
@@ -333,15 +333,19 @@ describe('pass domain into creator', () => {
     expect(fn).toHaveBeenCalledTimes(3)
   })
 
-  test('domain made from two parents should throw', () => {
+  test('unit made from two domains should select indirect one', () => {
+    const directFn = jest.fn()
     const direct = createDomain()
+    direct.onCreateStore(directFn)
+
+    const indirectFn = jest.fn()
     const indirect = createDomain()
+    indirect.onCreateStore(indirectFn)
 
-    expect(() => {
-      // @ts-ignore check for runtime error
-      direct.createDomain({domain: indirect})
-    }).toThrowErrorMatchingInlineSnapshot(
-      `"Domain cannot be created with two domains as parent. Please, remove {domain} argument."`,
-    )
+    // @ts-expect-error There is no types for case, just runtime check
```

Comment:
Probably the other way around 🤔 
`there is no runtime check for the case, only types`